core/src/main/java/io/questdb/griffin/SqlOptimiser.java

```diff
@@ -3801,6 +3801,90 @@ private ExpressionNode rewriteAggregate(ExpressionNode agg, QueryModel model) {
         return agg;
     }
 
+    /**
+     Rewrites
+     SELECT LAST(timestamp) FROM table_name;
+     into query which can is optimised search
+     SELECT timestamp from (SELECT timestamp from table_name order by timestamp) limit 1
+     This enables FrameBackwardScan for aggregation queries containing LAST keyword
+     */
+
```

Comment:
Due to this newline the above comment is floating: autoformatter may move it to a different location, away from the `optimiseAggregateFunctions` method.